.github/wiki/Fastlane.md

```diff
@@ -0,0 +1,161 @@
+# About
+
+Fastlane is a part of the automation tool for the development and release process of the project. This document will mention some crucial parts of the pipeline.
+
+<img src="assets/images/infrastructure/fastlane/fastlane.png" width=300>
+
+# Fastfile
```

Comment:
Let's change the level to `h2` ~ `##` instead, as the title of this document is `h1` already. Otherwise, the font size of this section will be the same with the title
```md
## Fastfile

### Build and test

### Synchronize the provisioning profile

...
## Matchfile
```